### ISSUE TITLE
Pims2MNT: Orthophotos not mosaicked

### ISSUE THREAD

**Issue by agonnaud** [micmacIGN/micmac#94]
Trying to extract a DSM and orthophotos from aerial imagery, I used the following chain:

OriConvert - Tapioca - Tapas - CenterBascule - Campari - AperiCloud - C3DC - Pims2MNT

Using almost exactly the same chain on the Saint Cloud drone demo data gave me satisfying results. However with my much larger dataset (photos at 7.5cm resolution) , I get a satisfying DSM but the orthophotos are not in a mosaic.

After running

mm3d Pims2MNT MicMac DoOrtho=1

the output directory contains a series of files such as the following:

-a----         7/5/2018   3:08 PM             41 Ort_09486Y16.tfw
-a----         7/5/2018   3:08 PM            282 Ort_09486Y16.tif
-a----         7/5/2018   3:13 PM      314573882 Ort_09486Y16_Tile_0_0.tif
-a----         7/5/2018   3:15 PM      314573882 Ort_09486Y16_Tile_0_1.tif
-a----         7/5/2018   3:13 PM       94372362 Ort_09486Y16_Tile_1_0.tif
-a----         7/5/2018   3:15 PM       94372362 Ort_09486Y16_Tile_1_1.tif
-a----         7/5/2018   3:11 PM             43 Ort_09487Y16.tfw
-a----         7/5/2018   3:11 PM            282 Ort_09487Y16.tif
-a----         7/5/2018   3:15 PM      314573882 Ort_09487Y16_Tile_0_0.tif
-a----         7/5/2018   3:17 PM      314573882 Ort_09487Y16_Tile_0_1.tif
-a----         7/5/2018   3:15 PM       94372362 Ort_09487Y16_Tile_1_0.tif
-a----         7/5/2018   3:17 PM       94372362 Ort_09487Y16_Tile_1_1.tif

The individual *Tile* files seem correct, but are obviously not georeferenced. The overall .tif files are extremely small (<1kb) and cannot be opened with QGIS. The tfw world files contain the georeferencing information for the overall tifs, not the Tiles.

Can anyone tell us what we are doing wrong ? Is this a memory issue ? Is this a bug ?

**Comment by agonnaud** [comment]
We just found out that we can merge the 4 tiles of a given .tif using ConvertIm. So we can work with the data. However, the question remains why Pims2MNT does not do this automatically.

**Comment by antochny** [comment]
Aren't you just missing Tawny command to create the orthomosaic ? or maybe I didn't get the problem...

**Comment by luc-girod** [comment]
> Aren't you just missing Tawny command to create the orthomosaic ? or maybe I didn't get the problem...

No, the Pims2MNT is doing the job, but the Ortho is too big for "standard" tiff files so it tiles the thing. Not sure why and where in the MicMac source this is coming from, but very big tiff are tilled. Merging is possible and straightforward.

**Comment by luc-girod** [comment]
> Aren't you just missing Tawny command to create the orthomosaic ? or maybe I didn't get the problem...

No, the Pims2MNT is doing the job, but the Ortho is too big for "standard" tiff files so it tiles the thing. Not sure why and where in the MicMac source this is coming from, but very big tiff are tilled. Merging is possible and straightforward.

**Comment by XRolland** [comment]
Thanks for this report @agonnaud. I experimented the same problem. Thanks for the workaround too.

Problem remains that, at the end of the process,  the ortho tif with its tfw remain **unusable** in GIS sofwtare without using ConvertIm. So it's a mess... 

Which solutions are possible?
- Is it possible to force tiles merge? Why is possible with ConvertIm as a workaround but not in the main process?
- Or, as a little workaround, could it be possible to add the tfw georef file  for each tile? So each tile could be usable?

**Comment by geodatup** [comment]
We are faced to the same problem and we would like to merge our 4 tiles with convertIm. But documentation is very poor about this tool and we can't figure out how to use it for the merge. Please @XRolland and @agonnaud could you tell more about this workaround ? thx

**Comment by geodatup** [comment]
I have my answer :  mm3d convertIm PIMs-ORTHO/Orthophotomosaic.tif do the job.
